packages/guideline-blocks-settings/src/styles.css

```diff
@@ -1,3 +1,5 @@
+@import '../node_modules/@frontify/fondue/dist/packages/components/style.css';
```

Comment:
Uff can't we do that in a nicer way?

Reply:
Copied from web-app 😅

I tried `@frontify/fondue-components/styles` but that did not work. Any other suggestions?

Reply:
Update: `@frontify/fondue/components/styles` seems to be working. That is indeed much nicer

Reply:
Love it 👍